packages/manager/apps/dedicated/client/app/ip/ip/agoraOrder/ip-ip-agoraOrder.constant.js

```diff
@@ -15,7 +15,34 @@ export const PRODUCT_TYPES = {
   },
 };
 
+export const IP_LOCATION_GROUPS = [
```

Comment:
You love making me update my PR about refactoring IP, don't you ?

Reply:
Haha, just wait for the final PR :D